starknet_py/net/account/_account_proxy.py

```diff
@@ -36,6 +42,17 @@ async def get_balance(
     ) -> int:
         return await self._account_client.get_balance(token_address=token_address)
 
+    async def sign_for_fee_estimate(
+        self, transaction: TypeAccountTransaction
+    ) -> TypeAccountTransaction:
+        version = self.supported_transaction_version + 2**128
```

Comment:
Are we sure we do not want to make it a constant value somewhere?
It's used in more than one place, in the same context, implicating a constant.

Reply:
This is an mostly an implementation detail. We could define a constant within Account maybe, but I'm not sure exposing it as a general constant in the library.

Reply:
Once we confirm that using `2**128` is fine for the purpose of non-executable transctions, let's reintroduce `FEE_VERSION_BASE = 2**128` constant and use it there.